resource/opentelemetry-resource-detector-container/tests/test_container.py

```diff
@@ -0,0 +1,120 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest.mock import mock_open, patch
+
+from opentelemetry import trace as trace_api
+from opentelemetry.resource.detector.container import ContainerResourceDetector
+from opentelemetry.sdk.resources import get_aggregated_resources
+from opentelemetry.semconv.resource import ResourceAttributes
+from opentelemetry.test.wsgitestutil import WsgiTestBase
```

Comment:
This is odd. Why does container resource detector package use the WSGI test base?

Reply:
WSGI server test is used to instrument a sample WSGI app and test if the container.id is present in the created span.

Reply:
Why do you need a WSGI server for that?

Reply:
@srikanthccv : any server test app would do. I just need a proper span in which I can see that container.id as a resource attribute. you can suggest if I can use something else here

Reply:
Resolved this issue and wsgi_app has been removed